### PR TITLE
SMP server CLI commands

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/smp-server/Main.hs [hunk 1]
```diff
@@ -173,8 +173,8 @@ fileExists path = do
   exists <- liftIO $ doesFileExist path
   unless exists . throwE $ "file " <> path <> " not found"
 
-deleteFileIfExists :: FilePath -> IO ()
-deleteFileIfExists path = doesFileExist keyPath >>= (`when` removeFile path)
+deleteIfExists :: FilePath -> IO ()
+deleteIfExists path = doesFileExist keyPath >>= (`when` removeFile path)
```
Comment (npoberezkin) [inline apps/smp-server/Main.hs:177]: should we maybe write:
`doesFileExist path >>= ...`

Comment (epoberezkin) [reply apps/smp-server/Main.hs]: fixed

File: apps/smp-server/Main.hs [hunk 2]
```diff
@@ -52,77 +52,129 @@ logDir = "/var/opt/simplex"
...
+          putStrLn "Server was initialized. Start it with `smp-server start` command"
+          printConfig cfg
+    ServerStart ->
+      runExceptT (getConfig opts) >>= \case
+        Right cfg -> runServer cfg
+        Left e -> do
+          putStrLn $ "Server is not initialized: " <> e
+          putStrLn "Initialize server with `smp-server init` command"
+          exitFailure
+    ServerDelete -> do
```
Comment (efim-poberezkin) [inline apps/smp-server/Main.hs:80]: can this be run at arbitrary moment, i.e. when server is started? if yes, will server just stop writing to store log after its deletion w/t crashing? if no, should there be a check that server is not started?

Comment (epoberezkin) [reply apps/smp-server/Main.hs:80]: I don’t hunk we should worry about it - it’s for the users who know why thaw doing.

**Review by npoberezkin — APPROVED** [review]

**Pull request merged**